org.tech.JTDX.yml

```diff
@@ -0,0 +1,63 @@
+app-id: org.tech.JTDX
+runtime: org.kde.Platform
+runtime-version: 5.15-22.08
+sdk: org.kde.Sdk
+command: jtdx
+rename-icon: jtdx
```

Comment:
it doesn't find an icon with this name.

Reply:
ok ! yes .. solving...

One question... I'm going to collaborate with more than 50 amateur radio programs

%99 uses hamlib .. my question is, wouldn't it be better to add a hamlib to the shared modules

to avoid having to indicate in so many programs to compile that library!

Reply:
this icon name doesn't match what's installed.